src/app/containers/SendInvoke/index.js

```diff
@@ -46,17 +50,18 @@ class SendInvoke extends Component {
     this.setState({ args: this.state.args.filter((s, idx) => id !== idx) })
   }
 
-  handleSubmit = event => {
+  _handleSubmit = event => {
```

Comment:
for consistency reasons, It'd be good practice to name all handlers with a `_` or without. Currently it's mixed